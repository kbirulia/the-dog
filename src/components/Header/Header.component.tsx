import React from "react";
import "./Header.style.css";
import { Link } from "react-router-dom";
import logo from "../../assets/img/logo.png";

const Header = (): JSX.Element => {
  return (
    <header className="header">
      <div className="main">
        <Link className="logo" to="/">
          <img
            width="70"
            height="70"
            src={logo}
            title="The dog"
            alt="The god"
          />
          <h1 className="logo__title">The dog</h1>
        </Link>
      </div>
    </header>
  );
};

export default Header;
