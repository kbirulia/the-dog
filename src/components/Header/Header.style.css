.header {
  background-color: #F05A14;
  height: 80px;
}

.logo {
  display: flex;
  align-items: center;
  text-decoration: none;
}

.logo__title {
  color: white;
  margin-left: 8px;
}

