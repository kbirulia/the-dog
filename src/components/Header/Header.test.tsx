import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import renderer from "react-test-renderer";
import Header from "./Header.component";

describe("Header", () => {
  test("should render correctly", () => {
    const tree = renderer
      .create(
        <Router>
          <Header />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
