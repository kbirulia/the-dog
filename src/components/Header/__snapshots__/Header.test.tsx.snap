// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<header
  className="header"
>
  <div
    className="main"
  >
    <a
      className="logo"
      href="/"
      onClick={[Function]}
    >
      <img
        alt="The god"
        height="70"
        src="logo.png"
        title="The dog"
        width="70"
      />
      <h1
        className="logo__title"
      >
        The dog
      </h1>
    </a>
  </div>
</header>
`;
